config/locales/en.yml

```diff
@@ -143,7 +143,7 @@ en:
 
   soft_validations:
     net_income:
-      hint_text: "You told us the main tenant's working situation is: <strong>%{ecstat1}</strong><br><br>The household income you have entered is <strong>%{earnings}</strong>"
+      hint_text: "<p class='govuk-body'>You told us the main tenant's working situation is: <strong>%{ecstat1}</strong></p><p class='govuk-body'>The household income you have entered is <strong>%{earnings}</strong></p>"
```

Comment:
What happens if you don’t add the classes?

```suggestion
      hint_text: "<p>You told us the main tenant's working situation is: <strong>%{ecstat1}</strong></p><p>The household income you have entered is <strong>%{earnings}</strong></p>"
```

Reply:
**With the class**
<img width="1088" alt="Screenshot 2022-03-10 at 16 53 36" src="https://user-images.githubusercontent.com/47317567/157714378-e9993557-6795-4a3f-b151-4b99233bf1aa.png">

**Without the class**
<img width="1070" alt="Screenshot 2022-03-10 at 16 54 10" src="https://user-images.githubusercontent.com/47317567/157714391-2bae64d0-6710-4e37-a925-218ce3113369.png">

Reply:
Ah, I’ll just make a quick tweak the CSS and add it to this PR…

Reply:
Tweaked!